keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/ApplicationControllerTests.kt

```diff
@@ -350,5 +342,13 @@ internal class ApplicationControllerTests : JUnit5Minutests {
           ))
       }
     }
+
+    testApiPermissions(mvc, jsonMapper, authorizationSupport, mapOf(
```

Comment:
what is this doing?

Reply:
https://github.com/spinnaker/keel/blob/2ac3cb709e91f0ca50c5773e8293d8e25fcefb0f/keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/ApplicationControllerTests.kt#L121-L142